management/sql/entities.go

```diff
@@ -71,6 +71,18 @@ type ServiceResource struct {
 	ServiceObjectiveID string `xml:"ServiceObjectiveId,omitempty"`
 }
 
+// ServiceResourceUpdateParams represents the set of parameters available
+// for a database service update operation.
+//
+// https://msdn.microsoft.com/en-us/library/azure/dn505718.aspx
+type ServiceResourceUpdateParams struct {
```

Comment:
move this below or up because currently ServiceResource and ListDatabasesResponse are related and they probably better sit next to each other.